src/components/pages/Offers/Offer/OfferLayout.jsx

```diff
@@ -138,7 +138,7 @@ const OfferLayout = ({ location, match }) => {
             exact
             path={`${match.url}/stocks`}
           >
-            <StocksContainer
+            <StocksGenericContainer
```

Comment:
```suggestion
            <Stocks
```

`Container` est un terme spécifique chez nous (ie un fichier qui va permettre de se brancher sur le router ou sur le store redux

Reply:
Découper au niveau de stockItem c'est effectivement à faire.
J'ai préféré le faire par itération plutôt que d'entrer dans tunnel de refacto.
Par contre je pense que de ne rien faire en attendant le big bang refacto, ca n'arrive jamais, on s'enfonce toujours plus dans le code compliqué et l'accroissement de la dette.